### PR TITLE
Updated en.gravatar.com links to gravatar.com

### PR DESCRIPTION
**Pull request by narenin**
Trac ticket: https://core.trac.wordpress.org/ticket/61424

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-admin/includes/upgrade.php [hunk 1]
```diff
@@ -257,13 +257,13 @@ function wp_install_defaults( $user_id ) {
...
 		$first_comment        = ! empty( $first_comment ) ? $first_comment : sprintf(
-			/* translators: %s: Gravatar URL. */
+			/* translators: %s: The localized Gravatar URL. */
 			__(
 				'Hi, this is a comment.
 To get started with moderating, editing, and deleting comments, please visit the Comments screen in the dashboard.
 Commenter avatars come from <a href="%s">Gravatar</a>.'
 			),
-			esc_url( __( 'https://en.gravatar.com/' ) )
+			esc_url( __( 'https://gravatar.com/' ) )
```
Comment (peterwilsoncc) [inline src/wp-admin/includes/upgrade.php:282]:
```suggestion
			/* translators: The localized Gravatar URL. */
			esc_url( __( 'https://gravatar.com/' ) )
```

File: src/wp-admin/user-edit.php [hunk 2]
```diff
@@ -621,9 +621,9 @@
 									<?php
 									if ( IS_PROFILE_PAGE ) {
 										$description = sprintf(
-											/* translators: %s: Gravatar URL. */
+											/* translators: %s: The localized Gravatar URL. */
```
Comment (peterwilsoncc) [inline src/wp-admin/user-edit.php]: As this refers to the placeholder, the translater comment can remain as is.

```suggestion
											/* translators: %s: Gravatar URL. */
```

File: src/wp-admin/user-edit.php [hunk 3]
```diff
@@ -621,9 +621,9 @@
...
 											__( '<a href="%s">You can change your profile picture on Gravatar</a>.' ),
-											__( 'https://en.gravatar.com/' )
+											__( 'https://gravatar.com/' )
```
Comment (peterwilsoncc) [inline src/wp-admin/user-edit.php:632]: The localization note can go in the URL. 

```suggestion
											/* translators: The localized Gravatar URL. */
											__( 'https://gravatar.com/' )
```

File: src/wp-admin/includes/upgrade.php [hunk 4]
```diff
@@ -257,13 +257,13 @@ function wp_install_defaults( $user_id ) {
 		$first_comment_email  = ! empty( $first_comment_email ) ? $first_comment_email : 'wapuu@wordpress.example';
 		$first_comment_url    = ! empty( $first_comment_url ) ? $first_comment_url : esc_url( __( 'https://wordpress.org/' ) );
 		$first_comment        = ! empty( $first_comment ) ? $first_comment : sprintf(
-			/* translators: %s: Gravatar URL. */
+			/* translators: %s: The localized Gravatar URL. */
```
Comment (peterwilsoncc) [inline src/wp-admin/includes/upgrade.php]:
```suggestion
			/* translators: %s: Gravatar URL. */
```

**Review by peterwilsoncc — COMMENTED** [review]
Add a couple of suggestions regarding the placement of the translator comments.

File: src/wp-admin/includes/upgrade.php (hunk 1 above)
Comment (narenin) [reply src/wp-admin/includes/upgrade.php:282]: @peterwilsoncc Thanks for the review I have implemented the suggestions.

File: src/wp-admin/user-edit.php (hunk 3 above)
Comment (narenin) [reply src/wp-admin/user-edit.php:632]: @peterwilsoncc Thanks for the review I have implemented the suggestions.

**Comment by narenin** [comment]
@peterwilsoncc Thanks for the review I have implemented the suggestions.

**Review by peterwilsoncc — APPROVED** [review]
Thanks, this looks good to me. I've asked a member of the polyglots team to take a look at the translator comments to validate them. Once that's done I will merge your code.

**Comment by peterwilsoncc** [comment]
Merged [r58810](https://core.trac.wordpress.org/changeset/58810) / https://github.com/WordPress/wordpress-develop/commit/4de5282f235f02605ffd86356c23d19c0a6b5ee7

**Pull request closed without merging**